src/components/SignUpForm/index.js

```diff
@@ -4,47 +4,83 @@ import "what-input";
 
 import "./SignUpForm.scss";
 
+import ENV from 'config/Environment';
+import MailchimpSubscribe from 'react-mailchimp-subscribe'
+
 class SignUpForm extends PureComponent {
   constructor(props) {
     super(...arguments);
 
     this.state = {
-      value: ""
+      emailAddress: '',
+      message: '',
     };
   }
 
   handleChange = event => {
-    this.setState({ value: event.target.value });
-  };
-
-  handleSubmit = event => {
-    event.preventDefault();
+    this.setState({ emailAddress: event.target.value });
   };
 
   render() {
+    const url = ENV.MAILCHIMP_URL;
```

Comment:
No need to store this in a const, as you're only using it once. You can just pass ```ENV.MAILCHIMP_URL``` directly into the url prop, or destructure it at the top of the file (after the import) like so: ```const { MAILCHIMP_URL } = ENV;```